core/gesture.ts

```diff
@@ -1139,6 +1139,11 @@ export class Gesture {
 
   /* End helper functions defining types of clicks. */
 
+  /** Returns the current dragger if the gesture is a drag. */
+  getCurrentDragger(): WorkspaceDragger | IDragger | null {
```

Comment:
I thought we weren't using this so I deleted it :P Turns out the scroll-options plugin needs it.